src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -403,7 +411,11 @@ object Parameters {
       }
     }
 
-    MergedParameters(DEFAULT_PARAMETERS ++ userParameters)
+    val parametersFromSnowpark = userParameters.get(PARAM_CONNECTION_ID) match {
+      case Some(id) => ServerConnection.providedConnections.getParameters(id)
+      case _ => Map.empty
+    }
```

Comment:
It will read `sf_url` and `sf_user` parameters from the provided JDBC connection.
Spark connector uses those values in the telemetry and log. We add those values here to avoid large amount of code change.